packages/hfs/src/proccessFiles.ts

```diff
@@ -0,0 +1,57 @@
+import { map, toArray } from 'rxjs/operators';
+import { transformSync } from '@babel/core';
+import babelRemoveImportsGetMeta from './babelRemoveImportsGetMeta';
+import md5 from 'md5';
+
+const tmpl = (arg, code) => `export default (obj) => {
+    ${arg !== '{}' ? 'const ' + arg + ' = obj;' : ''}
+    let exports = {};
+${code}
+    return exports;
+}
+`;
+const args = (info) => {
+  let ret = '';
+  info.source.forEach((i) => (ret = ret ? ret + ', ' + i.name : i.name));
+  return '{' + ret + '}';
+};
+
+const processFiles = (files) =>
+  files.pipe(
+    map((i: any) => {
+      const stats = { info: {} };
+      const transform = transformSync(i.content, {
+        code: true,
+        configFile: false,
+        plugins: ['@babel/plugin-syntax-dynamic-import', babelRemoveImportsGetMeta(stats)],
+        parserOpts: {
+          sourceType: 'module',
+        },
+      });
+      const code = transform ? transform.code : '';
+      const content = tmpl(args(stats.info), code);
+      const deps = {};
+      const dyDeps = {};
+      (stats.info as any).source.forEach((v, k) => {
+        if ((v as any).dynamic) {
+          dyDeps[k] = (v as any).name;
```

Comment:
call it with understandable name
```suggestion
       dynamicDeps[k] = (v as any).name;
```